src/main/resources/detect-sh.sh

```diff
@@ -27,7 +27,12 @@ DETECT_SOURCE=${DETECT_SOURCE:-}
 # *that* location will be used.
 # *NOTE* We currently do not support spaces in the
 # DETECT_JAR_DOWNLOAD_DIR.
-DETECT_JAR_DOWNLOAD_DIR=${DETECT_JAR_DOWNLOAD_DIR:-/tmp}
+if [ -z "${DETECT_JAR_DOWNLOAD_DIR}" ]; then
+	# If new name not set: Try old name for backward compatibility
+    DETECT_JAR_DOWNLOAD_DIR=${DETECT_JAR_PATH:-/tmp}
+else
+    DETECT_JAR_DOWNLOAD_DIR=${DETECT_JAR_DOWNLOAD_DIR:-/tmp}
+fi
 
```

Comment:
Perhaps move line 34 out of the else so we don't have two default locations to maintain?

Reply:
good idea; made that change